src/server/active_tran_server.hpp

```diff
@@ -34,6 +35,7 @@ class active_tran_server : public tran_server
 
     bool uses_remote_storage () const final override;
     MVCCID get_oldest_active_mvccid_from_page_server () const;
+    log_lsa compute_consensus_lsa () const;
```

Comment:
this function will drop the `const` upon #4135 because of the addition of the mutex

Reply:
Ok, removed in advance.